src/app/(private)/summary/page.tsx

```
@@ -5,13 +5,16 @@ import Link from 'next/link';
 import { BottomNav } from '@/components/layout/bottom-navigation';
 import { Header } from '@/components/layout/header';
 import { SummaryNavigateCard } from '@/components/summary/SummaryNavigateCard';
+import { useMe } from '@/hooks/auth/useMe';
 import { useSummaryList } from '@/hooks/summary/useSummaryList';
 import { useAppSelector } from '@/store/hooks';
```

ESLint

[error] 'useAppSelector' is defined but never used.

(unused-imports/no-unused-imports)